dlls/dinput/dinput_main.c

```diff
@@ -47,6 +47,7 @@
 #include "objbase.h"
 #include "rpcproxy.h"
 #include "initguid.h"
+#include "devguid.h"
```

Comment:
@aeikum just so you know, the line is already in the upstream and it may cause merge conflicts in the next rebase. I can cherry pick the upstreamed patches to the pull request to definitely avoid any merge issues if you wish.